### ISSUE TITLE
Tabs border rounding?

### ISSUE THREAD

**Issue by VitalSkib** [Guerra24/Firefox-UWP-Style#61]
Hi. 
I'm wondering if it's possible to add a slight tabs border rounding? 
Not so big, but kinda MS Edge style?
Thanks.

![edgelike](https://user-images.githubusercontent.com/43848892/173464775-eaeddda8-9182-4b0d-ae15-e7e27c290f70.png)

**Comment by Guerra24** [comment]
Hi.
There is a variant with rounded tabs, add `uwp.sun-valley.round-tabs` to enable it. WIP, has issues.
![image](https://user-images.githubusercontent.com/9023392/173466452-fc4e9d38-0190-4f81-ae2d-e9c8be69ee32.png)

**Comment by VitalSkib** [comment]
> Hi. There is a variant with rounded tabs, add `uwp.sun-valley.round-tabs` to enable it. WIP, has issues. ![image](https://user-images.githubusercontent.com/9023392/173466452-fc4e9d38-0190-4f81-ae2d-e9c8be69ee32.png)

Thanks for the quick answer, but no, this is not what I would prefer.. 
This, by the way, began to look like a new MS Edge style (image below), with Mica and rounded corners tabs.. 😃
But my 'goal' is the previous look, as on my image above.

![2022_06_14_Microsoft_Edge](https://user-images.githubusercontent.com/43848892/173469439-bfdd730c-731b-4b5c-b16a-68926d84ca91.png)

**Comment by VitalSkib** [comment]
Oh, if you don't mind, another small question, although not related to the main one. 
But just in order to not produce many new issues.
When I start Firefox, it shows its default window first (image below), while there are no styles loaded I guess... 
So, is there any way to replace it for any other image or just some solid color? 
Any ideas?
Their default purple tone everywhere is very annoying and that's why I love your theme/style which changes it almost everywhere. 😎👍
I wonder if Firefox has its own splash screen, hidden somewhere and could be enabled with some flags? 
It would help too, I think..

![2022_06_14_ff_startup](https://user-images.githubusercontent.com/43848892/173472915-f084c838-25ef-4781-8ad3-e894c94f009b.png)

**Comment by Guerra24** [comment]
> When I start Firefox, it shows its default window first (image below), while there are no styles loaded I guess...
So, is there any way to replace it for any other image or just some solid color?
Any ideas?

You can hide it with `browser.startup.preXulSkeletonUI`, afaik there are ways to customize it but those mess with the registry, at least on windows. Aside from that I don't know.

> Thanks for the quick answer, but no, this is not what I would prefer..

Sadly MS is *not that great* at design consistency (see tabbed explorer and edge) so for the time being I will go with the edge tabs since that is the style I already use in other stuff and I want, at least, some consistency.

**Comment by VitalSkib** [comment]
> You can hide it with `browser.startup.preXulSkeletonUI`, afaik there are ways to customize it but those mess with the registry, at least on windows. Aside from that I don't know.

Yes, it hides it, but now it is a white background flash instead, which is even worse, in my opinion.. 🤷‍♂️
But thanks, at least now I know that it can be turned off.. 

> Sadly MS is _not that great_ at design consistency (see tabbed explorer and edge) so for the time being I will go with the edge tabs since that is the style I already use in other stuff and I want, at least, some consistency.

By "I will go with the edge tabs" do you mean a latest one with separated tabs?
